web-common/src/features/dashboards/time-dimension-details/charts/TDDAlternateChart.svelte

```diff
@@ -34,6 +39,7 @@
   export let isTimeComparison: boolean;
 
   let viewVL: View;
+  let vegaSpec: any;
```

Comment:
Can we add the type for Vega Spec here? (should be `VisualizationSpec` or `VgSpec`)